cl/opinion_page/views.py

```diff
@@ -521,6 +522,14 @@ def view_opinion(request: HttpRequest, pk: int, _: str) -> HttpResponse:
         cluster,
     ).prefetch_related("representative",)[:3]
 
+    # Identify opinions updated/added in partnership with v|lex for 3 years
+    sponsored = False
+    if (
+        cluster.date_created.date() > datetime.datetime(2022, 6, 1).date()
+        and cluster.filepath_json_harvard
```

Comment:
This seems pretty good. Do you need the `.date()` casts? Why not just compare the datetime objects and have a little slop in the comparison?

Reply:
`can't compare offset-naive and offset-aware datetimes`

I thought it was easier then adding timezones